### ISSUE TITLE
Test 4.8

### ISSUE THREAD

**Issue by lmichel** [ivoa-std/ModelInstanceInVot#26]
This case tests a `TEMPLATES` without `@tableref`

The idea behind this is to allow this feature in the case we have only one  `TEMPLATES` .
In this case, `TEMPLATES` refers  to the first `TABLE`

- This rule cannot be set within the XSD (AFAIK), it must be stated in the document.
- The rationale for this shortcut is that this pattern matches most of the VOTables we can  found around the world. For instance, TAP services return one-table responses by construction.

**Comment by lmichel** [comment]
Let's take an example

```xml
<RESOURCE type="results">
     <RESOURCE type="meta">
         <VODML>
             <MODEL ... />
             <GLOBALS />
             <TEMPLATES>
                 <INSTANCE ... />
             </TEMPLATES
         </VODML>
     </RESOURCE>

    <TABLE ID="whatever">
        .......
    </TABLE>
 </RESOURCE>
```

This pattern (one table) matches almost all circulating VOTables. Allowing a tableref-less TEMPLATES when we have only one table simplify the job for the annoters since the same component can be used whatever the table name is.

This feature has been lately added to MiVot on the Vizier demand.

**Comment by lmichel** [comment]
tagged as valid

**Comment by mcdittmar** [comment]
I also recall that there were several native serializations where the TABLE did not have an ID, and I needed to add one to make a valid annotated file.  So, we're specifying that  each TEMPLATE => 1 TABLE, and if 'tableref' is missing, this implies the first TABLE block.

I'm not sure what the document text structure is looking like at this point, but may I suggest something like this for the descriptive text in the document/test snippet.

"
In many instances, the annotation is satisfied by a single TEMPLATES node associated with the, usually one and only, native TABLE block.  Additionally, we have found that in existing datasets, this TABLE block will often not have an assigned ID.  And so, in order to:
    * minimize the need to modify the native serialization
    * minimize the burden on annotators to explicitly define a self-evident relation
we allow that if the tableref is missing, the TEMPLATES block refers to the first TABLE.
"
